src/interfaces/vault.ts

```diff
@@ -583,7 +624,7 @@ export class VaultInterface<T extends ChainId> extends ServiceInterface<T> {
   }
 
   private async zapIn(
-    vault: Address,
+    toVault: Address,
```

Comment:
not sure if renaming is necessary here, since the function is `zapIn` so the `vault` will always be a to vault.

Reply:
The idea was to differentiate between vault object and vault address. I renamed both `toVault` and `fromVault` to `vaultAddr`.